src/controllers/midi/midicontroller.cpp

```diff
@@ -284,38 +284,42 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
 
     double newValue = value;
 
-
-    bool mapping_is_14bit = mapping.options.fourteen_bit_msb ||
-            mapping.options.fourteen_bit_lsb;
+    bool mapping_is_14bit = mapping.options.testFlag(MidiOption::FourteenBitMSB) ||
```

Comment:
Does this work?

`mapping.options.testFlag(MidiOption::FourteenBitMSB | MidiOption::FourteenBitLSB)`

Reply:
Nope, this does not work.